compose/project.py

```diff
@@ -197,23 +198,24 @@ def start(self, service_names=None, **options):
             service.start(**options)
 
     def stop(self, service_names=None, **options):
-        for service in reversed(self.get_services(service_names)):
-            service.stop(**options)
+        return self.container_executor("stop", service_names, **options)
```

Comment:
These `return`, but it looks like `container_executor()` doesn't return anything. Should it return the container list, or can we remove this return statement?